### ISSUE TITLE
Can´t find variable: PushNotification with 1.9.0 plugin version

### ISSUE THREAD

**Issue by 1989TomE** [phonegap/phonegap-plugin-push#1390]
Hello everybody, 

I am developing Cordova iOS app in Visual studio on Windows PC. I am deploying app to device via remote agent on my MAC. 

When I deploy app to a device I got following error in Web Inspector:
**Reference error: Can´t find variable: PushNotification**

**Project settings:**
Cordova CLI: 6.4.0
iOS platform version: 4.3.0
Plugin version: 1.9.0
Device: iPhone 5s (iOS 10.1.1)

I got this in my config.xml:

```
 <engine name="ios" spec="~4.1.0" />

<plugin name="phonegap-plugin-push" spec="https://github.com/phonegap/phonegap-plugin-push.git">
       <variable name="SENDER_ID" value="123456789xxxx" />
</plugin>
```

**Sample Push Data Payload**
$body['aps'] = array(
       'alert' => array(
	 'title' => $mobile_title,
	  'body' => $message,
	),
    'sound' => 'default',
    'badge' => 1, 
    );

**Sample Code that illustrates the problem:**
   var push = PushNotification.init({
        android: {
            senderID: "123456789xxxx",
        },
        ios: {
            alert: "true",
            badge: "true",
            sound: "true",
        },
        windows: {}
    });

I have been stuck for few days now. I didn´t install Cocoa Pods (Don´t know if this is optional or required?). Trick mentioned here https://github.com/phonegap/phonegap-plugin-push/issues/779 doesn´t seem to work now.

Also plugin was working fine with earlier versions before iOS 10 came out. 

Any help would be mostly appreciated. 

Tomas

**Comment by macdonst** [comment]
@1989TomE Yeah, [cocoapods are required](https://github.com/phonegap/phonegap-plugin-push/blob/master/docs/INSTALLATION.md#installation-requirements) in 1.9.0 and later. 

Try installing cocoapods or using 1.8.4.

**Comment by Dineshrajaa** [comment]
I faced the same issue in Android while building it in build.phonegap.com, downgrading it to v1.8.4 solved the issue
p.s: It is not happening while building it in local machine for android.

**Comment by 1989TomE** [comment]
Thanks you for your answers. 

I finally managed to resolved the issue by moving my cordova project to MAC. I upgraded cordova iOS platform to 4.3.0 there, than opened and deployed the app directly in Xcode. I installed cocoapods and pod file was created automatically (by upgrading platform or when building project I am not quite sure). 

Downgrading to v1.8.4 might be bettter solution regardless, because I wouldn´t have to move my project to MAC. Thanks for the tip.
